zap/src/test/java/org/zaproxy/zap/utils/JsonUtilUnitTest.java

```diff
@@ -139,6 +139,50 @@ void shouldNotQuoteFunctionHeader() {
         assertValueAfterJsonObject(processedValue, value);
     }
 
+    @Test
+    void shouldReturnKeyStrForJsonObject() {
+        // Given
+        String json =
+                """
```

Comment:
Put the block start on the same line as the = ?

Reply:
I mean `        String json = """` like it is on L160.

Reply:
Yeah, I did that and then ran `splotlessApply` and it changed it back 😁 Maybe because the line is longer?